site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -455,6 +455,20 @@ This means you are attempting to attach a Policy to a Route that has an overlapp
 - Combine the Route rules for the overlapping path into a single Route.
 - If the Policy allows it, specify both Routes in the `targetRefs` list.
 
+##### Broken Header error
+
+If you check your _nginx_ container logs and see the following error:
+
+```text
+  2024/07/25 00:50:45 [error] 211#211: *22 broken header: "GET /coffee HTTP/1.1" while reading PROXY protocol, client: 127.0.0.1, server: 0.0.0.0:80
+```
+
+It indicates that `proxy_protocol` is enabled for the gateway listeners, but the request sent to the application endpoint does not contain proxy information. To **resolve** this, you can do one of the following:
+
+- Disable field [`rewriteClientIP.mode`](({{< relref "reference/api.md" >}})) in the NginxProxy configuration.
```

Comment:
This wording is a bit misleading because there isn't a way to disable the mode. Maybe unset?

Reply:
I used `unassign` , hope that works